deploy/helpers.py

```diff
@@ -35,6 +35,8 @@ def remove_quotes(string):
     warehousename = remove_quotes(config.get(f"connections.{profile}", "warehousename"))
     dbname = remove_quotes(config.get(f"connections.{profile}", "dbname"))
 
+    if len(dbname) == 0:
+        raise ValueError("Database must be specified in the config string.")
```

Comment:
put the config string name instead. More descriptive

Reply:
done